Makefile

```diff
@@ -15,6 +15,7 @@ LINT_FLAG        = --eval "(setq byte-compile-dest-file-function (lambda (f) (co
 TEST_COMMAND     = buttercup -L $(SRC_DIR) -L $(EXTRA_DIR) $(NO_LOAD_WARNINGS)
 
 ELS  = $(SRC_DIR)/treemacs.el
+ELS  = $(SRC_DIR)/treemacs-all-the-icons.el
```

Comment:
Never mind, copyright's not quite the last thing. This line needs to use `+=` and `SRC_DIR` and should be moved down to the other addons in line 50.